web/models/stable_diffusion/main.py

```diff
@@ -162,3 +184,25 @@ def stable_diff_inf(
     text_output += f"\nTotal image generation time: {total_time:.4f}sec"
 
     return pil_images[0], text_output
+
+
+if __name__ == "__main__":
```

Comment:
This won't work due to two different entry points with different relative imports (index.py and models/stable_diffusion/main.py)
```
Traceback (most recent call last):
  File "/home/quinn/nod/SHARK/web/models/stable_diffusion/main.py", line 5, in <module>
    from models.stable_diffusion.cache_objects import (
ModuleNotFoundError: No module named 'models'
```
This is why we need #694 first to enable absolute imports for all of the relevant SD code.

Reply:
@qedawkins Let's land #694 first then. Thanks!

Reply:
Ok, so I have removed this second entry point from this PR as it is already there in #694. Now this PR is good to go before #694. I will take a look at the merging stuff with #694.